detect_secrets/plugins/common/initialize.py

```diff
@@ -12,6 +12,7 @@
 from ..keyword import KeywordDetector                       # noqa: F401
 from ..private_key import PrivateKeyDetector                # noqa: F401
 from ..slack import SlackDetector                           # noqa: F401
+from ..artifactory import ArtifactoryDetector               # noqa: F401
```

Comment:
Nit: can you put this on line 7, so they're alphabetical

Reply:
@KevinHock nit addressed!